src/clip.sol

```diff
@@ -196,10 +200,10 @@ contract Clipper is LibNote {
         // Could get this from rmul(Vat.ilks(ilk).spot, Spotter.mat()) instead,
         // but if mat has changed since the last poke, the resulting value will
         // be incorrect.
-        (PipLike pip, ) = spot.ilks(ilk);
+        (PipLike pip, uint256 mat) = spot.ilks(ilk);
```

Comment:
The "liquidation price" is just defined as `debt/collateral` at liquidation time, it shouldn't include `mat`.

Reply:
Although I can see the argument for correcting for it: it makes the price higher in the case of a phony oracle price; the tradeoff is that you risk setting the price too high in real declines. Deciding which is better (without being able to see the future) requires having beliefs about the likelihood and severity of various scenarios; this might be an excellent thing to have Gauntlet take a look at.

Reply:
My opinion is that it is better to have the higher of the two (`include mat`) because since this is just the initial price of the dutch auction, the price discovery can still occur later on after some decrease in the scenario of a sharp market decline. But yes that's a great point about Gauntet. I'm going to leave as is if no one has any major objections and we can revisit during their review. @kmbarry1 if you think its better that I take out `mat` before the Gauntlet review I can do that.

Reply:
Let's leave it in for now; at a minimum we should get an opinion from someone who works on risk. There's a call with Gauntlet tomorrow, I'll bring it up there.

Reply:
Perfect 👍  sounds good.